### PR TITLE
[15.0] [MIG] stock_quant_reservation_info: Migration to 15.0

### PR DESCRIPTION
**Pull request by bizzappdev**

### CODE REVIEW

File: stock_quant_reservation_info/models/stock_quant.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ class StockQuant(models.Model):
     def action_reserved_moves(self):
         self.ensure_one()
         action = {
-            "name": _("Reserved moves for: " + self.product_id.name),
+            "name": _("Reserved Moves for: %s") % self.product_id.name,
```
Comment (rousseldenis) [inline stock_quant_reservation_info/models/stock_quant.py]: `_("Reserved Moves for: %(product_name)s", product_name=self.product_id.name)` is better

**Review by rousseldenis — CHANGES_REQUESTED** [review]

File: stock_quant_reservation_info/models/stock_quant.py (hunk 1 above)
Comment (bizzappdev) [reply stock_quant_reservation_info/models/stock_quant.py]: Changes are done

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review LGTM

**Review by DavidJForgeFlow — APPROVED** [review]
Code LGTM! Also it works fine!

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**